TYPO3.Neos/Documentation/CreatingASite/ContentCache.rst

```diff
@@ -135,6 +139,11 @@ In the ``@cache`` meta property the following subproperties are allowed:
 		}
 	}
 
+``entryDiscriminator``
+  Configure an expression that uniquely discriminates different entries of a ``dynamic`` cached area. The expression or TypoScript
+  object must evaluate to a string to be used as discriminator and should be different for every cache entry you want to create for
+  this ``dynamic`` cached area.
+
 .. _Cache Entry Tags:
 
```

Comment:
Not the fault of this patch, but I think to truly understand how the different caching options are used, we really need some short real world examples with working TypoScript snippets.